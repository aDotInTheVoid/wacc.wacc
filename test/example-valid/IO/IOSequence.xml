<unit>
  <main>
    <addr_local>0</addr_local>
    <num>0</num>
    <assign_do/>
    <str>"Please input an integer: "</str>
    <print><kind>str</kind><newline>false</newline></print>
    <addr_local>0</addr_local>
    <read><ty>int</ty></read>
    <str>"You input: "</str>
    <print><kind>str</kind><newline>false</newline></print>
    <local>0</local>
    <print><kind>i32</kind><newline>true</newline></print>
  </main>
</unit>