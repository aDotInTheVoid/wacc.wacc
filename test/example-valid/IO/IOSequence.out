Please input an integer: You input: 42
